crates/chia-consensus/src/gen/conditions.rs

```diff
@@ -82,9 +83,31 @@ impl SpendVisitor for MempoolVisitor {
 
     fn condition(&mut self, spend: &mut SpendConditions, c: &Condition) {
         match c {
-            Condition::AssertMyCoinId(_) => {
+            Condition::AssertMyCoinId(_)
+            | Condition::AssertHeightRelative(_)
+            | Condition::AssertSecondsRelative(_)
+            | Condition::AssertBeforeHeightRelative(_)
+            | Condition::AssertBeforeSecondsRelative(_)
+            | Condition::AssertMyBirthHeight(_)
+            | Condition::AssertMyBirthSeconds(_)
+            | Condition::AssertEphemeral => {
```

Comment:
`ASSERT_EPHEMERAL`, `ASSERT_BEFORE_SECONDS_RELATIVE` and `ASSERT_BEFORE_HEIGHT_RELATIVE` should have tests too. you could try to comment out one at a time and make sure the tests fail